core/src/test/java/io/questdb/griffin/LimitTest.java

```diff
@@ -224,6 +224,24 @@ public void testLastN() throws Exception {
         testLimit(expected, expected2, query);
     }
 
+    @Test
+    public void testLimitMinusOneAndPredicateAndColumnAlias() throws Exception {
+        compiler.compile("create table t1 (ts timestamp, id symbol)", sqlExecutionContext);
+
```

Comment:
Not an issue but you can use insert into with multiple rows to remove repetition : 

```sql
insert into t1 values (0L, 'abc'), (2L, 'a1'), ... 
```

Reply:
thanks. to was a copy'n'paste from some existing test. let me change that.